website/src/pages/index.js

```diff
@@ -261,22 +261,23 @@ const NewsletterSection = () => {
 const PicPoster = () => {
   const [display, setDisplay] = useState(true);
   const [theme, setTheme] = useState('light');
+  const windowGlobal = typeof window !== 'undefined' && window
```

Comment:
Just do window.getItem inside useEffect instead of this. This will always put undefined on windowGlobal at build as window is not defined when SSR.

Reply:
![image](https://user-images.githubusercontent.com/72343596/125797734-15d3fcd8-99e1-47dd-b84d-6eaea2cf4d6a.png)
Hi, what should we do with the localStorage that is not on the hook?

Reply:
I guess one solution could be to use useRef hook and put it on the button, then target event of click inside useEffect. Like classic ref.current.addEventListener()